letsencrypt/client.py

```diff
@@ -329,8 +329,23 @@ def deploy_certificate(self, domains, privkey_path,
                 self.installer.save()  # needed by the Apache plugin
 
             self.installer.save("Deployed Let's Encrypt Certificate")
-            # sites may have been enabled / final cleanup
+
+        # sites may have been enabled / final cleanup
+        with error_handler.ErrorHandler(self._rollback_and_restart):
+            self.installer.restart()
+
```

Comment:
At this point, print "Rollback successful; Apache has been restarted with your old configuration."